src/js/views/patients/sidebar/flow/flow-sidebar_views.js

```diff
@@ -107,7 +107,7 @@ const SidebarView = View.extend({
   modelEvents: {
     'change:name': 'showName',
     'change:details': 'showDetails',
-    'change:_state': 'showOwner',
```

Comment:
Not sure why this was `showOwner`? May be missing something, but I think `showFlow` is what this should be.

Reply:
canEdit is only affected by the owner. I think this should just be `'change:_state': 'showState'`